server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.monitor.metrics;
+
+import org.elasticsearch.action.admin.cluster.node.stats.NodeStats;
+import org.elasticsearch.action.admin.indices.stats.CommonStatsFlags;
+import org.elasticsearch.common.util.SingleObjectCache;
+import org.elasticsearch.core.TimeValue;
+import org.elasticsearch.monitor.jvm.GcNames;
+import org.elasticsearch.monitor.jvm.JvmStats;
+import org.elasticsearch.node.NodeService;
+import org.elasticsearch.telemetry.metric.LongWithAttributes;
+import org.elasticsearch.telemetry.metric.MeterRegistry;
+
+import java.util.Map;
+
+public class NodeMetrics {
+    private final NodeService nodeService;
+    private final NodeStatsCache nodeStatsCache;
+
+    public NodeMetrics(MeterRegistry meterRegistry, NodeService nodeService) {
+        this.nodeService = nodeService;
+        this.nodeStatsCache = new NodeStatsCache(TimeValue.timeValueSeconds(1));
+        registerMetrics(meterRegistry);
+    }
+
+    private void registerMetrics(MeterRegistry registry) {
+        registry.registerLongGauge(
+            "es.node.stats.indices.get.total",
+            "Total number of get operations",
+            "operation",
+            () -> new LongWithAttributes(nodeStatsCache.getOrRefresh().getIndices().getGet().getCount(), Map.of())
+        );
+        registry.registerLongGauge(
+            "es.node.stats.indices.get.time_in_millis",
```

Comment:
I was looking at this PR as an example of how to do metrics and also looking at the "Metrics Naming Guideline Proposal" doc. I noticed that the doc says not to include units in the metric name but instead put them in the unit. So should "in_millis" be removed from the name here and the unit changed to "millis"? (Or alternatively, if this PR is best practice then the doc should be changed to match.)